.rubocop.yml

```diff
@@ -49,3 +49,12 @@ RSpec/SpecFilePathSuffix:
 
 RSpec/VerifiedDoubleReference:
   Enabled: false
+
+RSpec/LetSetup:
```

Comment:
I'm guessing you need this for the `let!` in spec where you are performing actions before the shared example.

Reply:
You can avoid this by moving the `let!` declarations into the shared examples file. Which should be fine, because they're actually identical between both sets of system specs.

Then we shouldn't need this rule exclusion.